content-moderation/02-video-moderation/01-content-moderation-video.ipynb

```diff
@@ -78,14 +80,16 @@
     "s3=boto3.client('s3', region_name=region)\n",
     "rekognition=boto3.client('rekognition', region_name=region)\n",
     "sagemaker = boto3.client('sagemaker', region_name=region)\n",
-    "comprehend = boto3.client('comprehend', region_name=region)"
+    "comprehend = boto3.client('comprehend', region_name=region)\n",
+    "transcribe=boto3.client('transcribe', region_name=region)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "For this lab, we will moderate a demo video containing a few typical inappropriate scenes, such as alcohol, tobacco, gambling, and suggestive. The video locates at `datasets/moderation-video.mp4`.\n",
+    "For this lab, we will moderate a demo video containing typical inappropriate scenes, such as alcohol, tobacco, and suggestive. The video also has text and audio containing inappropriate information, such as profanity. You can find the video at `datasets/moderation-video.mp4`.\n",
```

Comment:
What is meant by "suggestive"?

My best guess is "sexually suggestive", but could there be terms that are more objective such as "nudity" or "sexual activity"? (I'm taking inspiration from the likes of https://www.bbfc.co.uk/rating/12 and https://www.filmratings.com/RatingsGuide).

At the least, "suggestive" is ambiguous and should be elaborated upon.

Reply:
"Suggestive" is a Rekognition predefined Content Moderation category.